docs/api/structures/protocol-request.md

```diff
@@ -0,0 +1,6 @@
+# ProtocolRequest Object
+
+* `url` String
+* `referrer` String
+* `method` String
+* `uploadData` [UploadData[]](upload-data.md)
```

Comment:
The request won't always have an `uploadData` property right? Or will it be an empty array when it isn't a POST request?

```suggestion
* `uploadData` [UploadData[]](upload-data.md) (optional)
```